README.md

```diff
@@ -1 +1,28 @@
-# Hello-World
\ No newline at end of file
+# Hello-Microverse
+A simple "Hello Microverse" page
+## Built With
+
```

Comment:
- [ ] [OPTIONAL] Kindly add a screenshot of your project so that it looks more professional and, so you can get used to it for other Microverse projects.